JournalMonitor/JournalMonitor.cs

```diff
@@ -2594,7 +2594,7 @@ async void ModuleArrived()
                                     decimal total = JsonParsing.getDecimal(data, "Total");
                                     bool full = EDDI.Instance.CurrentShip?.fueltanktotalcapacity == null
```

Comment:
Let's introduce a variable for `EDDI.Instance.CurrentShip` to improve readability.